Traceback (most recent call last):
  File "/home/ifam/fischer/Code/MORe_DWR/Heat/ROM/2D/slabwise/pod_rom_error.py", line 26, in <module>
    [data, row, column] = np.loadtxt(OUTPUT_PATH + cycle + "/matrix_no_bc.txt")
  File "/home/ifam/fischer/.local/lib/python3.6/site-packages/numpy/lib/npyio.py", line 961, in loadtxt
    fh = np.lib._datasource.open(fname, 'rt', encoding=encoding)
  File "/home/ifam/fischer/.local/lib/python3.6/site-packages/numpy/lib/_datasource.py", line 195, in open
    return ds.open(path, mode, encoding=encoding, newline=newline)
  File "/home/ifam/fischer/.local/lib/python3.6/site-packages/numpy/lib/_datasource.py", line 535, in open
    raise IOError("%s not found." % path)
OSError: ../../../Data/2D/rotating_circle/slabwise/cycle=6/matrix_no_bc.txt not found.
